sorc/ncep_post.fd/ALLOCATE_ALL.f

```diff
@@ -1,1295 +1,1312 @@
-        allocate(ocsmass(im,jsta_2l:jend_2u))
-        allocate(occmass(im,jsta_2l:jend_2u))
-        allocate(ocsmass25(im,jsta_2l:jend_2u))
-        allocate(occmass25(im,jsta_2l:jend_2u))
-        allocate(bcsmass(im,jsta_2l:jend_2u))
-        allocate(bccmass(im,jsta_2l:jend_2u))
-        allocate(bcsmass25(im,jsta_2l:jend_2u))
-        allocate(bccmass25(im,jsta_2l:jend_2u))
-        allocate(sssmass(im,jsta_2l:jend_2u))
-        allocate(sscmass(im,jsta_2l:jend_2u))
-        allocate(sssmass25(im,jsta_2l:jend_2u))
-        allocate(sscmass25(im,jsta_2l:jend_2u))
-        allocate(dustcb(im,jsta_2l:jend_2u))
-        allocate(occb(im,jsta_2l:jend_2u))
-        allocate(bccb(im,jsta_2l:jend_2u))
-        allocate(sulfcb(im,jsta_2l:jend_2u))
-        allocate(pp25cb(im,jsta_2l:jend_2u))
-        allocate(pp10cb(im,jsta_2l:jend_2u))
-        allocate(sscb(im,jsta_2l:jend_2u))
-        allocate(dustallcb(im,jsta_2l:jend_2u))
-        allocate(ssallcb(im,jsta_2l:jend_2u))
-        allocate(dustpm(im,jsta_2l:jend_2u))
-        allocate(sspm(im,jsta_2l:jend_2u))
-!Initialization
-!$omp parallel do private(i,j)
-       do j=jsta_2l,jend_2u
-         do i=1,im
-           dusmass(i,j)=spval
-           ducmass(i,j)=spval
-           dusmass25(i,j)=spval
-           ducmass25(i,j)=spval
-           susmass(i,j)=spval
-           sucmass(i,j)=spval
-           susmass25(i,j)=spval
-           sucmass25(i,j)=spval
-           ocsmass(i,j)=spval
-           occmass(i,j)=spval
-           ocsmass25(i,j)=spval
-           occmass25(i,j)=spval
-           bcsmass(i,j)=spval
-           bccmass(i,j)=spval
-           bcsmass25(i,j)=spval
-           bccmass25(i,j)=spval
-           sssmass(i,j)=spval
-           sscmass(i,j)=spval
-           sssmass25(i,j)=spval
-           sscmass25(i,j)=spval
-           dustcb(i,j)=spval
-           occb(i,j)=spval
-           bccb(i,j)=spval
-           sulfcb(i,j)=spval
-           pp25cb(i,j)=spval
-           pp10cb(i,j)=spval
-           sscb(i,j)=spval
-           dustallcb(i,j)=spval
-           ssallcb(i,j)=spval
-           dustpm(i,j)=spval
-           sspm(i,j)=spval
-         enddo
-       enddo
-      endif
-! HWRF RRTMG output 
-      allocate(acswupt(im,jsta_2l:jend_2u))
-      allocate(swdnt(im,jsta_2l:jend_2u))
-      allocate(acswdnt(im,jsta_2l:jend_2u))
-!Initialization
-!$omp parallel do private(i,j)
-      do j=jsta_2l,jend_2u
-        do i=1,im
-          acswupt(i,j)=spval
-          swdnt(i,j)=spval
-          acswdnt(i,j)=spval
-        enddo
-      enddo
-
-! UPP_MATH MODULE DIFFERENTIAL EQUATIONS
-      allocate(ddvdx(im,jsta_2l:jend_2u))
-      allocate(ddudy(im,jsta_2l:jend_2u))
-      allocate(uuavg(im,jsta_2l:jend_2u))
-!Initialization
-!$omp parallel do private(i,j)
-      do j=jsta_2l,jend_2u
-        do i=1,im
-          ddvdx(i,j)=spval
-          ddudy(i,j)=spval
-          uuavg(i,j)=spval
-        enddo
-      enddo
-! 
-      end
+!> @file

```

Comment:
@hsinmulin-NOAA Please check out a fresh version of ALLOCATE_ALL.f from upstream/develop and incorporate your changes. Your current version modify the format of the file so it is hard to track your actual changes.

Reply:
@hsinmulin-NOAA I know you only add small part of changes, Somehow, it is a long list  in this file. So people are hard to see your actual changes.